datadog_cluster_agent/assets/dashboards/datadog_cluster_agent_overview.json

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
Is there a link to the dashboard synced on staging? Easier to review 😄

Reply:
Yes there is! [Here](https://ddstaging.datadoghq.com/dashboard/har-9wi-8iz/juliogreffdca?from_ts=1595487838213&live=true&to_ts=1595491438213). Title and description were changed from the json (I haven't synced it back to staging after small changes), but everything else is the same.

Reply:
Awesome, I've cloned that dashboard and made some edits that would make it align more with our preset dashboard guidelines. See: https://ddstaging.datadoghq.com/s/c894ecec5/tsm-g8i-fxw

Here is some more information on Datadog's best practices and style guide: https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices

Reply:
I just incorporated the edits. Thank you!